source/views/settings/screens/api-test/api-test-android.js

```diff
@@ -65,16 +65,6 @@ export class AndroidAPITestView extends React.PureComponent<Props, State> {
 	render() {
 		let {error, results} = this.state
 
-		let APIResponse = error ? (
```

Comment:
as another note… capitalized variables should be reserved for … er, component creation functions? Constructors and etc. 

APIResponse, I would expect to need `<APIResponse/>`, whereas this would be `{APIResponse}`.

I prefer and try to keep the code to `{apiResponse}` for this sort of "data" variable.

Just a side note.

Reply:
noted, thanks!